models/user.go

```diff
@@ -65,6 +72,17 @@ func (d *UserData) Value() interface{} {
 		return d.NumericValue
 	case BOOL:
 		return d.BoolValue
+	case MAP:
+		data := &map[string]interface{}{}
+		json.Unmarshal([]byte(d.SerializedValue), data)
+		return data
+	case ARRAY:
```

Comment:
I was thinking we could pull out some key ones we want - specifically roles for now. and have a table that is just `user_id:role`, doesn't solve the generic case but /shrug